### DISCUSSION TITLE
Regarding running the game

### DISCUSSION THREAD

**Discussion by freakun0025** [LN-Studios/Matchup-League#7] [Q&A]
from the pom.xml I am unable to run the error i am facing is 
[ERROR] Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:3.3.5:run (default-cli) on project matchup: Process terminated with exit code: 1 -> [Help 1]
Please help urgently needed

**Comment by lognog2** [comment]
See my response to the other discussion
